index.md

```diff
@@ -4,61 +4,38 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
 ---
 
-# Why should I take this course?
+This workshop is specifically aimed at running the [LAMMPS](https://lammps.sandia.gov/) software on a HPC system. You may be running LAMMPS on either a desktop, laptop or already on an HPC system, however ineffective use of LAMMPS can lead to running jobs which could take far too long. Using a HPC system effectively can speed up your LAMMPS runs significantly and vastly improve its performance if given the right information, which we aim to cover in this workshop.
 
->Before you start the class, type a few sentences about why you are here.
+Some questions that you may ask yourself are;
 
-At this point, I assume that all of you have some level of experiences of running [LAMMPS](https://lammps.sandia.gov/) software on a HPC system. A HPC enable us to run jobs that would either be impossible or much slower either in a Desktop or a laptop. The phrase 'much slower' in the previous sentence is not much informative. The first question that arises here is:
->   * How slow is running LAMMPS on my HPC than its standard performance?
+* What is meant by the term 'performance' of a software?
+* What is the metric of measuring this performance?
+* How do I measure standard performance of a software?
+* How slow is LAMMPS running on my HPC relative to its standard performance?
+* **If a software performance is not optimal in my system, is there something that can I do to accelerate it?**
```

Comment:
`optimal` is a dangerous word, there are too many factors at play, `acceptable` is a better option since it acknowledges that there is probably always something we could do a little better. There is a good concept that comes from the VI-HPS workshops, which (para-phrasing) is: 80% of the performance improvement comes with 20% of the effort, that last 20% may not be worth it, know when to stop.